corehq/apps/performance_sms/dbaccessors.py

```diff
@@ -23,9 +21,9 @@ def by_interval(interval_keys):
 def delete_all_configs():
     assert settings.UNIT_TESTING
     db = PerformanceConfiguration.get_db()
-    all_docs = [row['doc'] for row in db.view(
-        'performance_sms/by_domain',
-        reduce=False,
-        include_docs=True
-    )]
-    db.bulk_delete(all_docs)
+
+    # Since this is a unit test, number of domains is small
```

Comment:
To @czue's note, I agree there's a better way to delete all `PerformanceConfiguration` docs. In fact, such a function exists! See `delete_all_docs_by_doc_type`.